quality/finder/finder_hyper.inst.cfg

```diff
@@ -0,0 +1,57 @@
+[general]
+version = 4
+name = Standard Resolution
+definition = finder
+
+[metadata]
+setting_version = 13
+type = quality
+quality_type = hyper
+weight = 0
+global_quality = True
+
+[values]
+layer_height = 0.08
+layer_height_0 = 0.20
+
+top_layers = 7
+bottom_layers = 3
+adhesion_type = skirt
+speed_slowdown_layers = 1
+travel_retract_before_outer_wall = True
+cool_fan_full_layer = 1
+cool_min_layer_time = 8
+cool_min_speed = 15
+expand_skins_expand_distance = 0.1
+infill_enable_travel_optimization = True
+infill_pattern = zigzag
+material_flow = 109
+meshfix_maximum_resolution = 0.05
+optimize_wall_printing_order = True
+retract_at_layer_change = True
+retraction_amount = 1.3
+retraction_combing_max_distance = 2
+retraction_speed = 30
+skin_preshrink = 0.1
+speed_print = 80
```

Comment:
For 80 Microns , we dont want to print at 80mm/s maybe around 20-25mm/s that will give best result. Do you have .curaprofile of these profiles? if so i can quickly change values and send you back.

Reply:
Oh, sorry, i just copied the file to show other resolutions. You are correct this will not look good.